faq.yaml

```diff
@@ -54,33 +53,34 @@ en:
         msg.value greater than 0 for the entirety of the 3, 6 or 12-month lock durations”,
         and in these cases “that signal can be treated as a lock for the purposes of the allocation award”. 
 
-        By removing the ability to signal on behalf of a contract, Straightedge has solved for Generalized
-        Locks and other edge case vulnerabilities. However, it is still worth explicitly pointing out some
+        By removing the ability to signal on behalf of a contract, Straightedge has ntended to reduce vulnerabilities by
+        simplifying the distribution mechanism.  However, it is still worth explicitly pointing out some
         of the issues with the policy. First, it places undue burden upon the launchers of the network to
         audit and decide on different generalized lock contracts, which may contain edge cases such as
         bypassing locking by issuing derivatives of the locked Ether (such as with WETH). Further, if a
         contract does not correctly lock, or if code claiming to be a lock bypasses detection through
         underhanded Solidity techniques, it could break the legitimacy of the initial distribution of
-        the entire network. Removing generalized locking removes the necessity for case-by-case decision-making
-        by a launch-team, and other edge case vulnerabilities.
+        the entire network. While Edgeware’s contract codes have been fully audited by Quantstamp,  Removing generalized
```

Comment:
Typo: unnecessary extra space between Quantstamp, and Removing

Typo: Removing should be without capital.

Reply:
The Commonwealth-written contracts aren't relevant to this point.  It's about new generalized lock contracts.  Is Quantstamp going to audit all of those?